plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -751,7 +751,17 @@ jQuery( function ( $ ) {
 					}
 				);
 				selectedAttributes = newSelectedAttributes;
-				$( 'select.wc-attribute-search' ).data('disabled-items', newSelectedAttributes );
+				$( 'select.wc-attribute-search' ).data(
+					'disabled-items',
+					newSelectedAttributes
+				);
+				var isUsedForVariations = $( 'input#used-for-variation' ).is(
+					':checked'
+				);
+				$( 'select.wc-attribute-search' ).data(
```

Comment:
I think this selector is only available if there are no attributes globally already available.

This could also be `attribute_taxonomy`.

https://github.com/woocommerce/woocommerce/blob/98a82c018cff47419dc7811bcaf6d685721da3f5/plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-attributes.php#L40